src/global-settings.php

```diff
@@ -541,8 +595,8 @@ public function create_style( $style, $value ) {
 		public function generate_typography_styles( $selector, $styles ) {
 			$inherit = true;
 			$inherit_max = 50;
-			$tablet_breakpoint = 1025;
-			$mobile_breakpoint = 768;
+			$tablet_breakpoint = 1023;
+			$mobile_breakpoint = 767;
```

Comment:
Was this change intentional? Typography styles will change
```suggestion
			$tablet_breakpoint = 1025;
			$mobile_breakpoint = 768;
```